kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -63,6 +63,11 @@
   public static final String REGION_CONFIG = "s3.region";
   public static final String REGION_DEFAULT = Regions.DEFAULT_REGION.getName();
 
+  public static final String S3_PROXY_HOST = "s3.proxy.host";
```

Comment:
I'm a bit torn between having two properties, or just adding `S3_PROXY` and (eventually?) support all the properties in one string, as described here: http://docs.aws.amazon.com/cli/latest/userguide/cli-http-proxy.html (the rest are protocol, username and password, and a couple more on windows ...)

In that case though, it seems that protocol should be supported from the start (or maybe just accept `host` or `host:port` for now, and later add support for `protocol://username:pass@host:port` and valid combinations)